keyboards/monsgeek/m2/info.json

```
@@ -0,0 +1,271 @@
+{
+    "keyboard_name": "M2",
+    "manufacturer": "MonsGeek",
+    "maintainer": "jonylee@hfd",
+    "usb": {
+        "vid": "0xFFFE",
+        "pid": "0x0008",
+        "device_version": "1.0.4",
+        "suspend_wakeup_delay": 400,
+        "force_nkro": true
+    },
+    "processor": "WB32FQ95",
+    "bootloader": "wb32-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": false,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["C1", "C2", "C3", "A0", "A1", "A2", "A3", "A4", "A5", "A6", "A7", "C4", "C5", "B0", "B1", "B2", "B10", "B11", "B12", "B13", "B14"],
+        "rows": ["B15", "C6", "C7", "C8", "C9", "A8"]
+    },
+    "diode_direction": "ROW2COL",
+    "indicators": {
+        "num_lock": "C10",
+        "caps_lock": "A15"
+    }, 
```

migrated from `config.h`
```suggestion
    },
    "dynamic_keymap": {
        "layer_count": 6
    },
```